script.js

```diff
@@ -0,0 +1,164 @@
+      1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry.
+      Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type
+      and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.
+      Lorem Ipsum has been the industry standard dummy text ever since the 1500s,
+      when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+    mobiledesc: `Lorem Ipsum is simply dummy text of the printing and typesetting industry
+      Lorem Ipsum has been the industry standard dummy text ever since the 1500s,
+       when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+      It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.`,
+    link: 'https://kevin-mena.github.io/Personal-Portfolio-Website/',
+    source: ' https://github.com/Kevin-Mena/Personal-Portfolio-Website/tree/popup-window',
+  },
+];
+const extra = document.querySelector('#extra');
+const project = document.querySelectorAll('#expand');
+function display(main, i) {
+  const play = `<div class="content">
+      <div class="top">
+        <h2 class="mobile">${main[i].mobileTopic}</h2>
+        <h2 class="desktop">${main[i].desktopTopic}</h2>
+        <ul>
+          <li class="dark desktop cl">Uber</li>
+          <li class="dark mobile">canopy</li>
+          <li class="desktop">full Stack Dev</li>
+          <li class="mobile">Back End Dev</li>
+          <li class="desktop">2018</li>
+          <li class="mobile">2015</li>
+        </ul>
+       </div>
+      <div class="image">
+        <img src="${main[i].mobileImage}"  class="mobile"/>
+        <img src="${main[i].desktopImage}" class="desktop"/>
+      </div>
+      <div class="bottom">
+        <div class="words">
+          <p class="desktop">
+            ${main[i].desktopdesc}
+          </p>
+          <p class="mobile">
+            ${main[i].mobiledesc}
+          </p>
+        </div>
+        <div class="other">
+          <ul class="stack">
+            <div class="stt">
+            <li>html</li>
+            <li>javascript</li>
+            <li>css</li>
+            </div>
+            <li class="desktop">ruby </li>
+            <li class="desktop">github</li>
+            <li class="desktop">bootstrap</li>
+          </ul>
+          <div class="button">
+            <a href="${main[i].link}" class="btn">
+              see live
+              <img src="img/seelive.png" alt="visit the website"/>
+            </a>
+            <a href="${main[i].source}" class="btn">
+              see source
+              <img src="img/seesource.png" alt=" see the sourcecode"/>
+            </a>
+          </div>
+        </div>
+        <img src="img/cancel.png" alt="To close panel" id="cancel-port"/>
+      </div>`;
```

Comment:
- [ ] Firstly, I must say that your project is awesome overall 👏  You really took time to give a great design. However, there's a slight variation between the design of your skills and source links to your project, with what's presented in the figma design. The source button that directs to the `live` version and project `source code` does not show up in the mobile view. It's visible on the desktop view, and perfectly set, and I strongly suggest you make a similar design on the mobile. You'd agree with me that this gives a clear path for web accessibility for mobile users and gives your potential viewers paths to authenticate your project from the source and have a clear web view of what it looks like.

- [ ] Note that at the initial load, the buttons are present, but they don't render the view is rendered. kindly check the view below 💯 

# Your present view:
![image](https://user-images.githubusercontent.com/50990925/215968249-556059e0-1e7f-44e9-8eb8-48c239ad2a05.png)

# View expected from figma design :
![image](https://user-images.githubusercontent.com/50990925/215968315-079f1c41-47fa-49ab-b758-0f9a8fb2358e.png)